redfish-exporter/listener.go

```diff
@@ -185,10 +187,16 @@ func processRequest(AppConfig Config, conn net.Conn, req *http.Request, eventCou
 		log.Printf("Message Args: %v", messageArgs)
 		log.Printf("Origin Of Condition: %s", originOfCondition)
 		for _, triggerEvent := range AppConfig.TriggerEvents {
-			if eventType == triggerEvent.EventType && eventId == triggerEvent.EventId && severity == triggerEvent.Severity {
+			redfishServerIP := fmt.Sprintf("https://%v", remote)
+			if eventType == triggerEvent.EventType && eventId == triggerEvent.EventId {
 				log.Printf("Matched Trigger Event: %s with action %s", triggerEvent.EventId, triggerEvent.Action)
-				// TODO: Add the SLURM integration here
 				// Sending event belings to redfish_utils. Each server may have different slurm node associated, and redfish_servers has the info/map.
+				for _, redfishServer := range AppConfig.RedfishServers {
+					if redfishServer.IP == redfishServerIP {
+						performEventAction(triggerEvent.Action, redfishServer.SlurmNode)
```

Comment:
This will do for confirming the functionality and so on. Eventually, we can decouple the `AppConfig` from the listener, and the `performEventAction` could be replaced here to pushEvents to a slurm_queue, allowing us to process them separately. as well as the event mapping which was also discussed here:
https://github.com/nod-ai/ADA/pull/8#issuecomment-2393432352

Reply:
I’ve implemented a basic queue to handle the processing of these actions. In the future, we can enhance it by creating a dedicated queue for each Redfish server, or implement other optimizations as needed. Please take a look and share your feedback.

Reply:
LGTM. Nice work!